host/cluster-down.yaml

```diff
@@ -0,0 +1,306 @@
+# Spec to run when a kURL cluster is down and in-cluster specs can't be run
+apiVersion: troubleshoot.sh/v1beta2
+kind: SupportBundle
+metadata:
+  name: cluster-down
+spec:
+  hostCollectors:
+    # System Info Collectors
+    - blockDevices: {}
+    - cpu: {}
+    - hostOS: {}
+    - hostServices: {}
+    - ipv4Interfaces: {}
+    - memory: {}
+    - time: {}
+    # Certificate Info for ETCD and K8s API
+    - certificate:
+        collectorName: k8s-api-keypair
+        certificatePath: /etc/kubernetes/pki/apiserver.crt
+        keyPath: /etc/kubernetes/pki/apiserver.key
+    - certificate:
+        collectorName: etcd-keypair
+        certificatePath: /etc/kubernetes/pki/etcd/server.crt
+        keyPath: /etc/kubernetes/pki/etcd/server.key
+    # Disk usage for commonly used directories in kURL installs
+    - diskUsage:
+        collectorName: root
+        path: /
+    - diskUsage:
+        collectorName: var-lib-kubelet
+        path: /var/lib/kubelet
+    - diskUsage:
+        collectorName: var-lib-docker
+        path: /var/lib/docker
+    - diskUsage:
+        collectorName: var-lib-containerd
+        path: /var/lib/containerd
+    - diskUsage:
+        collectorName: var-lib-rook
+        path: /var/lib/rook
+    - diskUsage:
+        collectorName: opt-replicated
+        path: /opt/replicated
+    - diskUsage:
+        collectorName: var-openebs
+        path: /var/openebs
```

Comment:
I also saw a case where `/tmp` was 2GB and too small to create a support bundle for the cluster - might be a good one to add here.

Reply:
Makes sense

Reply:
Actually, do you have any details on what the requirement/suggestion should be here? And did this happen when generating from the CLI or from kotsadm? And was it `/tmp` in the Pod or on the host?

Reply:
[this comment](https://github.com/replicated-collab/puppet-kots/issues/194#issuecomment-1071188063) suggests that 2GB for `/tmp` is too small,  as troubleshoot itself uses /tmp as a location to construct the bundle.  I can't confirm without testing if that's just from the CLI or from kots also.  It was `/tmp` on the host, so this would be the sort of thing that prevents collecting a bundle rather than necessarily preventing BAU operation, however /tmp being too small is an issue for many applications and maybe we should recommend 8GB at least as a starting point.